src/tests.c

```diff
@@ -2967,7 +2967,6 @@ static int check_fe_equal(const secp256k1_fe *a, const secp256k1_fe *b) {
     secp256k1_fe an = *a;
     secp256k1_fe bn = *b;
     secp256k1_fe_normalize_weak(&an);
-    secp256k1_fe_normalize_var(&bn);
```

Comment:
Not sure why `normalize_var` was used on `bn`. I assumed it intended to set `bn`'s magnitude to 1 for the `fe_equal` call. So, I removed it. Please let me know if that's not the case, and I'll revert this. 

All the tests pass with this change.